### PR TITLE
Use the new, shorter link for downloading the install script

### LINKED ISSUE #2427: Auto-magic shell one-liner installer
**Is your feature request related to a problem? Please describe.**
A one-liner installation option makes for a great README for the users, e.g.:

    ```bash
    $ curl -fsSL https://get.pulumi.com/ | sh
    ```

Real world example: https://github.com/pulumi/pulumi-java/tree/main#getting-started

**Describe the solution you'd like**
A shell script hosted at `https://get.scala-cli.virtuslab.org/` that automatically detects the version required by the user.

**Describe alternatives you've considered**
An alternative is to not do it and keep the 3-liner installation.

### PR DESCRIPTION
**Pull request by Gedochao**
Fixes #2427 

We can now use the shorter link to get the install script: https://scala-cli.virtuslab.org/get

### CODE REVIEW

File: modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala [hunk 1]
```diff
@@ -522,26 +522,30 @@ trait RunScriptTestDefinitions { _: RunTestDefinitions =>
   test("script wrappers satisfy strict compiler flags") {
     val inputs = TestInputs(
       os.rel / "strictClassWrapper.sc" ->
-      """//> using scala 3.3.1
```
Comment (tgodzik) [inline modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala:525]: I am fine with the formatting changes, but curious why this happend.

**Review by tgodzik — APPROVED** [review]

File: modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala (hunk 1 above)
Comment (Gedochao) [reply modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala:525]: I merely ran `./mill -i __.fix && scalafmt`
curious indeed

Comment (Gedochao) [reply modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala:525]: CI format checks suceeded, so I guess it was unformatted before? how come 🤔

Comment (MaciejG604) [reply modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala:525]: Some looking into our format check is required, this is not the first time it happens :(

**Review by MaciejG604 — APPROVED** [review]

**Comment by pawelprazak** [comment]
thank you! :)

**Pull request merged**